cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -45,10 +45,12 @@ def normalize_paths(paths)
       (is_extension && path.match('/'))
       raise ArgumentError.new("Invalid path: #{path}")
     end
+    single_path = !path.match('\*')
```

Comment:
I did a refactoring pass on this part of the code, should make this code a little bit more straightforward (at the expense of making the underlying generated VCL a bit more verbose).